sprites/frog/frogCharacter.js

```diff
@@ -2,6 +2,7 @@
 const frogCharacter = {
   name:"frogCharacter",
   size: {width: 228, height: 288},
+  animationTypes: ['IDLE', 'WALK', 'EAT', 'CELEBRATE', 'DISGUST', 'ALL'],
```

Comment:
Is there a reason we aren't going for a name for the walk-on animation that is standard across all characters?